mlir/include/mlir/Dialect/Affine/Passes.td

```diff
@@ -199,7 +199,7 @@ def AffineLoopTiling : Pass<"affine-loop-tile", "func::FuncOp"> {
   ];
 }
 
-def AffineLoopUnroll : Pass<"affine-loop-unroll", "func::FuncOp"> {
+def AffineLoopUnroll : Pass<"affine-loop-unroll", "ModuleOp"> {
```

Comment:
This isn't the right change.

Make it an InterfacePass on FunctionOpInterface

Reply:
Thanks you, I think I learned something new.